tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -2354,27 +2354,42 @@ public function userGetsAllSharesSharedWithHimFromFileOrFolderUsingTheProvisioni
 		);
 	}
 
-	/**
-	 * @When user :user gets all shares shared by him/her using the sharing API
-	 *
+	/***
 	 * @param string $user
+	 * @param string|null $endpointPath
 	 *
 	 * @return void
+	 * @throws JsonException
+	 * @throws GuzzleException
 	 */
-	public function userGetsAllSharesSharedByHimUsingTheSharingApi(string $user):void {
+	public function getAllShares(
+		string $user,
+		?string $endpointPath = null
+	): void {
 		$user = $this->getActualUsername($user);
 		$this->response = OcsApiHelper::sendRequest(
 			$this->getBaseUrl(),
 			$user,
 			$this->getPasswordForUser($user),
 			"GET",
-			$this->getSharesEndpointPath(),
+			$this->getSharesEndpointPath($endpointPath),
 			$this->getStepLineRef(),
 			[],
 			$this->ocsApiVersion
 		);
 	}
 
+	/**
+	 * @When user :user gets all shares shared by him/her using the sharing API
```

Comment:
```suggestion
	 * @When user :user gets all shares shared by (?:him|her) using the sharing API
```

can we do like this?

Reply:
It won't find step with `shared by him`. Probably write `(him?:him|her)`?